### DISCUSSION TITLE
Reqnroll Sponsorship Model Draft

### DISCUSSION THREAD

**Discussion by gasparnagy** [reqnroll/Reqnroll#61] [Announcements]
I have created a draft for the Reqnroll sponsorship model. Please have a look and give feedback. I would like to publish it to the website in a few weeks.

We accept sponsorship in various different forms, but particularly:
1.	Recurring or one-time contribution as money
2.	Offering workforce, by offering fixed or recurring man-hours of work from the employees of the sponsoring company
3.	Offering licenses or other benefits for the key contributors of the project

Regardless of the form of the contribution, we always calculate the yearly value of the sponsorship in EUR to make them comparable. For workforce, we use a fixed rate of 80 EUR / h for the calculation; for licenses the yearly license price (calculated based on 3 years of usage).

Based on the yearly value, we imagined 4 sponsorship tiers:

* **Supporter**
  * Yearly contribution: below €200
  * Benefits
    *  The name of the supporter is listed on the website’s sponsoring page and in GitHub README.md
* **Silver**
  * Yearly contribution: between €200 and €5000
  * Benefits
    * The name & the logo of the sponsor is listed on the website’s sponsoring page as link; 
    * 3 randomly selected silver sponsor is listed on the project home page (re-randomized for every page load); 
    * The name & link of the sponsor is listed in the GitHub README.md (without logo).
    * If the sponsor has a tool that integrates with Reqnroll, it can be listed in the “Integrations” section of the documentation.
* **Gold**
  * Yearly contribution: between €5000 and €20 000
  * Benefits
    * The benefits of Silver tier, plus
    * The sponsor is always visible on the website homepage 
    * The sponsor is included in README.md with logo
    * A short promo text (max 100 words) can be included in the sponsor page
    * The sponsor can provide 1 guest blog post per year that is published on the Reqnroll website and promoted on social channels
* **Platinum**
  * Yearly contribution: above €20 000
  * Benefits
    * The benefits of Gold tier, plus
    * Extra highlight of the sponsorship level in all lists (listed as platinum sponsor before gold and silver sponsors; bigger logo)
    * Handling of special sponsor requests 

It is not only related to the sponsorship model, but probably worth listing here how we would use the money we have collected. Again, just draft.

The money we collect from the sponsors (or the profit of the paid subscription packages) are spent to the following goals:
* Operational costs: website hosting, domain fee, additional build agents, administration costs, etc. (expected: <€2000 year)
* Marketing: producing and distributing marketing material, like stickers, t-shirts, etc. (expected <€2000 year)
* Contributor connections: Helping key contributors to meet (e.g. supporting them to attend a conference and meet other contributors) (expected: <€20 000 / year)
* Paid workforce for special feature implementations: hire experts for helping with concrete tasks (budget: ad hoc)
* Contributor compensation: Pay compensation for open source contributors for their work (budget: as lasts, target: €80/h gross)

Please let me know if something is unclear or any other feedback.

**Comment by SabotageAndi** [comment]
There is a small typo. I think Gold should start at €5000 and not €500, or?

**Reply by gasparnagy** [reply]
Fixed, thx

**Comment by 304NotModified** [comment]
https://github.com/sponsors/reqnroll links to this discussion. I think it should link to https://reqnroll.net/sponsorship/ nowadays?

**Reply by 304NotModified** [reply]
Polite bump @gasparnagy 

(Maybe we could also close this)

**Reply by gasparnagy** [reply]
Fixed. The go.reqnroll.net redirects can now edited via the https://github.com/reqnroll/website.go/blob/main/site/_redirects repo/file now (migrated a few weeks ago). If you see any other "wrong" target there, please send a PR.

**Comment by ahmedalejo** [comment]
Awesome effort @gasparnagy and team! It´s clear and reasonable!
